.github/workflows/adoc_build.yml

```diff
@@ -1,7 +1,8 @@
 name: Export and publish document
 
 on:
-  workflow_dispatch:
```

Comment:
I suggest keeping this one together with push. I found it useful when developing to be able to recompile it manually. By keeping both, we expect to have the auto mode on daily activities.